lib/money/accounting_money_parser.rb

```diff
@@ -1,7 +1,11 @@
-class AccountingMoneyParser < MoneyParser
-  private
-  def extract_money(input)
-    # set () to mean negativity. ignore $
-    super(input.gsub(/\(\$?(.*?)\)/, '-\1'))
+class AccountingMoneyParser < DeprecatedMoneyParser
+  class << self
```

Comment:
it's a personal taste, so is `class << self` some kinda pattern used inside `money` gem?

I personally think that when we have 1-2 (few) class methods, I'd rather have `def self.my_method` than using `class << self`. I think this way you have a better visualization.

Again this is a personal taste 👅

Reply:
@filipebarcos I don't think you can do true private class methods without `class << self`

Reply:
RIGHT! sorry, missed the `private`